atc/engine/engine.go

```diff
@@ -421,18 +421,12 @@ func (c *engineCheck) clearRunState() {
 
 func (c *engineCheck) trackStarted(logger lager.Logger) {
 	metric.CheckStarted{
-		CheckName:             c.check.Plan().Check.Name,
-		ResourceConfigScopeID: c.check.ResourceConfigScopeID(),
-		CheckStatus:           c.check.Status(),
-		CheckPendingDuration:  c.check.StartTime().Sub(c.check.CreateTime()),
+		CheckStatus: c.check.Status(),
```

Comment:
I don't think we need to include this field; it'll always be `started` which is kind of implied by the metric already.